### ISSUE TITLE
app.js fails to run on Win10 with local Vagrant based Fabric

### ISSUE THREAD

**Issue by LeonardHarley** [IBM-Blockchain/car-lease-demo#36]
Hi - when running node app.js....

`C:\Code\go_code\src\github.com\ibm-blockchain\car-lease-demo>node app.js
String 'http://127.0.0.1'
Updated config file.
------------------------------------------ Server Up - http://localhost:80 ------------------------------------------
ENV VARIABLES http://127.0.0.1 5000
ENTER   Startup INPUT: {}
INFO    Startup Locating initial participants

I0924 19:50:37.337000000  3928 subchannel.c:642] Connect failed: null
I0924 19:50:37.338000000  3928 subchannel.c:645] Retry immediately
I0924 19:50:37.339000000  3928 subchannel.c:609] Failed to connect to channel, retrying
INFO    Startup Failed to register using HFC, user may have already been enrolled. Error

INITIAL LOGIN ATTEMPT undefined
ERROR   Startup OUTPUT: "[object Object] Error: chain registrar is not set"`

**Comment by iamsaurabhc** [comment]
Did you get any solution? I'm also stuck at the same place! I've used Docker in a Local VM
![screenshot from 2016-11-10 01-42-59](https://cloud.githubusercontent.com/assets/19235748/20153073/0df933f4-a6e7-11e6-9fa2-5702b9e9b0be.png)

**Comment by lenh** [comment]
No. If I recall correctly the car lease demo is older code running on a very specific version of the Fabric, Node SDK etc etc...

**Comment by 14gracel** [comment]
The car lease demo has now been ported to use Fabric 0.6. See issue #59 if trying to deploy locally still. Please close this issue if the problem has been solved, thanks

**Comment by bhaskarbanerjee** [comment]
Trying the latest version of app https://github.com/IBM-Blockchain/car-lease-demo
Deployment of app from the bluemix page https://console.ng.bluemix.net/docs/services/blockchain/ibmblockchain_tutorials.html#carlease works but any subsequent redeployment of the same app (locally cloned) using CLI results in the following message in the app logs.
Thereafter all admin/identity services fail and everything falls apart.

App/0ERROR	Startup	OUTPUT: "Failed to enroll registrar with WebAppAdmin 2dc560a51d"06 Dec 2016 | 11:28 AM
App/0ERROR	Startup	OUTPUT: {"code":2,"metadata":{"_internal_repr":{}}}06 Dec 2016 | 11:28 AM
App/0 at /home/vcap/app/node_modules/grpc/src/node/src/client.js:417:17 code: 2, metadata: Metadata { _internal_repr: {} } }06 Dec 2016 | 11:28 AM
App/0{ Error: Identity or token does not match.06 Dec 2016 | 11:28 AM
App/0ENTER	GET blockchain/blocks	INPUT: ""

Any clues?
